Use the old cmake provided FindProtobuf() implementation for now.
A future cmake update will remove this module, but use it for now until
all packages are updated to the new system
---
--- a/cmake/protobuf.cmake	2023-06-14 22:21:16.000000000 +0200
+++ -	2023-07-10 21:05:41.114443796 +0200
@@ -55,7 +55,7 @@
     set(gRPC_INSTALL FALSE)
   endif()
 elseif(gRPC_PROTOBUF_PROVIDER STREQUAL "package")
-  find_package(Protobuf REQUIRED CONFIG)
+  find_package(Protobuf REQUIRED)
 
   if(Protobuf_FOUND OR PROTOBUF_FOUND)
     if(TARGET protobuf::${_gRPC_PROTOBUF_LIBRARY_NAME})
